agent/ui_endpoint.go

```diff
@@ -771,6 +771,7 @@ func (s *HTTPHandlers) UIMetricsProxy(resp http.ResponseWriter, req *http.Reques
 		Director: func(r *http.Request) {
 			r.URL = u
 		},
+		Transport: s.proxyTransport,
```

Comment:
If nil, Transport defaults to `http.DefaultTransport`: 
https://github.com/golang/go/blob/go1.19.1/src/net/http/httputil/reverseproxy.go#L52-L54